src/kitFilterHelpers.ts

```diff
@@ -0,0 +1,54 @@
+import { generateHash } from "./utils";
+// TODO: https://mparticle-eng.atlassian.net/browse/SQDSDKS-5381
+import { EventTypeEnum, IdentityType } from "./types.interfaces";
+import Constants from './constants';
+
+const { CCPAPurpose } = Constants;
+
+
+export default class KitFilterHelpers {
+    static hashEventType(eventType: EventTypeEnum): number {
+        return generateHash(eventType);
+    };
+
+    static hashEventName(eventName: string, eventType: EventTypeEnum): number {
+        return generateHash(eventType + eventName);
+    };
+
+    static hashEventAttributeKey(eventType: EventTypeEnum, eventName: string, customAttributeName: string): number {
+        return generateHash(eventType + eventName + customAttributeName);
+    }
+    
+    static hashUserAttributeKey(userAttributeKey: string): number {
+        return generateHash(userAttributeKey);
+    }
+
+    static hashUserAttributeValue(userAttributeValue: string): number {
+        return generateHash(userAttributeValue);
+    }
+
+    // User Identities are not actually hashed, this method is named this way to
+    // be consistent with the filter class. UserIdentityType is also a number
+    static hashUserIdentity(userIdentity: IdentityType): IdentityType {
+        return userIdentity;
+    }
+
+    static hashGDPRPurpose(purpose: string): number {
+        const GDPRConsentHashPrefix = '1';
+
+        return generateHash(GDPRConsentHashPrefix + purpose)
+    }
+
+    static hashCCPA(){
```

Comment:
Nitpick: For consistency, since we have `hashGDPRPurpose`, this should be `hashCCPAPurpose`

Reply:
[I actually originally had this, but I removed](https://github.com/mParticle/mparticle-web-sdk/pull/531/commits/9908880408d1062b2421ceedac07feacd39d5653) `purpose` because ccpa only has one purpose, and we pass a purpose into the GDPR, but it seems inconsistent to not pass a purpose into CCPA.  I could go either way.  input from @einsteinx2 and @markvdouw might be valuable as well.